src/AppCard.test.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import { render } from 'react-testing-library'
+
+import electionSample from './data/electionSample.json'
+
+import Root, { App } from './App'
+import { ActivationCardData, AdminCardData, Election } from './config/types'
+
+const election = electionSample as Election
+
+beforeEach(() => {
+  window.localStorage.clear()
+  window.location.href = '/'
+})
+
+it(`App fetches the card data every 1 second`, () => {
+  fetchMock.resetMocks()
+  jest.useFakeTimers()
+
+  fetchMock.mockResponses(
+    [JSON.stringify({}), { status: 200 }],
+    // This response covers the App functionality for processing card data.
```

Comment:
I added this comment because in this test file the data contained in the following response is not confirmed to have been received later in this same test… however it does satisfy code coverage.